packages/svelte-query/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@tanstack/svelte-query",
+  "version": "4.14.5",
+  "description": "Primitives for managing, caching and syncing asynchronous and remote data in Svelte",
+  "author": "Dre Johnson",
+  "license": "MIT",
+  "repository": "tanstack/query",
+  "homepage": "https://tanstack.com/query",
+  "funding": {
+    "type": "github",
+    "url": "https://github.com/sponsors/tannerlinsley"
+  },
+  "type": "module",
+  "module": "build/lib/index.js",
```

Comment:
do we not need a separate entry for the `types` that point to the `.d.ts` files?

Reply:
Added it now!